tst/regression/test_suites/restart/parthinput.restart

```diff
@@ -19,20 +19,20 @@ refinement = adaptive
 numlevel = 3
 
 nx1 = 64
-x1min = -0.5
-x1max = 0.5
-ix1_bc = periodic
-ox1_bc = periodic
+x1min = -1
+x1max = 1
+ix1_bc = outflow
+ox1_bc = reflecting
 
 nx2 = 64
-x2min = -0.5
-x2max = 0.5
+x2min = -1
+x2max = 1
```

Comment:
The restart test now runs the sparse advection example instead of the regular advection example. And the sparse advection example has a hard-coded assumption that the domain is [-1,1].